### PR TITLE
MS-158: Profiles Interviews Sessions Applications Final Form

### PR DESCRIPTION
**Pull request by andrespienizzio**

### CODE REVIEW

File: src/Components/Admin/Applications/Form/index.js [hunk 1]
```diff
@@ -1,150 +1,160 @@
...
+import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import { Form, Field } from 'react-final-form';
 import styles from './form.module.css';
 import Spinner from 'Components/Shared/Spinner';
+import Select from 'Components/Shared/Select';
 import Input from 'Components/Shared/Input';
 import Button from 'Components/Shared/Button';
 import { getOneApplication } from 'redux/Applications/thunks';
+import { cleanSelectedApplication } from 'redux/Appplications/actions';
```
Comment (BareaMartin) [inline src/Components/Admin/Applications/Form/index.js:10]: Here you have the folder applications with 3 p.

File: src/redux/Sessions/reducer.js [hunk 2]
```diff
@@ -16,7 +16,9 @@ import {
   GET_ONE_SESSION_REJECTED,
   SHOW_MODAL,
   SHOW_MESSAGE,
-  MODAL_TYPE
+  MODAL_TYPE,
+  CLEAN_ERROR,
+  CLEAN_SELECTED_ADMIN
```
Comment (BareaMartin) [inline src/redux/Sessions/reducer.js:21]: Here you forgot to change ADMIN, to SESSION

File: src/redux/Sessions/reducer.js [hunk 3]
```diff
@@ -157,7 +159,18 @@ const sessionsReducer = (state = initialState, action) => {
...
       };
     }
-
+    case CLEAN_ERROR: {
+      return {
+        ...state,
+        error: initialState.error
+      };
+    }
+    case CLEAN_SELECTED_ADMIN: {
```
Comment (BareaMartin) [inline src/redux/Sessions/reducer.js:168]: Here you have the same issue as before, it should be SESSION.

**Review by BareaMartin — CHANGES_REQUESTED** [review]

**Review by BareaMartin — APPROVED** [review]
tested while sharing screen on discord.

**Review by julianv97 — APPROVED** [review]

**Pull request merged**